### ISSUE TITLE
New Vivaldi version moves the address bar to bottom of tab bar

### ISSUE THREAD

**Issue by foosroos** [HKayn/vivaldi-vh#11]
Operating System: Windows 11
Vivaldi Version: 6.9.3447.37 (Stable channel) (64-bit) 
CSS Version: 1.4.1

Recently updated my browser and the address bar moved from the top of the panel down to the bottom.

![image](https://github.com/user-attachments/assets/9bf3a003-2a85-4a36-b2a4-315b6ebb4bff)

**Comment by HKayn** [comment]
Thanks for reporting this! This issue is already known and will be fixed in the next version of Vivaldi VH releasing in the next couple days.

Until then, you can apply a temporary fix by creating a new `.css` file in the same directory where you placed Vivaldi VH, and then pasting the following code into the new file:

```css
/* Vivaldi VH: Workaround for misplacement of the address bar in Vivaldi 6.9 */
#main .mainbar > * {
    display: contents;
}
```

Be advised that the name of the `.css` file must not contain any spaces.

**Comment by HKayn** [comment]
This is now fixed with the latest release v1.4.2.

Commit [966d873](https://github.com/HKayn/vivaldi-vh/commit/966d873310030cf4250d88f8c4af70656fe1790f) closes this.
